docs/guides/storybook-npm-setup.md

```diff
@@ -0,0 +1,155 @@
+
+# Storybook - npm setup - draft
+_Based on PR [#135](https://github.com/bbc/react-transcript-editor/pull/135)_
+
+## webpack 
+Webpack is now used for bundling the component for npm distribution. 
+Biggest change is that the build script now uses webpack instead of simply using babel
```

Comment:
Leave out previous process out of this doc if it's not mentioned anywhere else

Reply:
If you do describe the previous process, you may want to describe why you changed it.

Reply:
That's an ADR here https://github.com/bbc/react-transcript-editor/blob/master/docs/adr/2018-10-05-component-development-setup.md